src/views/darkroom.c

```diff
@@ -2239,6 +2282,47 @@ void gui_init(dt_view_t *self)
                               G_CALLBACK(_display2_profile_changed), (gpointer)display2_profile);
   }
 
+  /* create overlay color changer popup tool */
+  {
+    // the button
+    dev->overlay_color.button
+        = dtgtk_togglebutton_new(dtgtk_cairo_paint_grid, CPF_STYLE_FLAT, NULL);
+    gtk_widget_set_tooltip_text(dev->overlay_color.button,
+                                _("set the color of lines that overlay the image (drawn masks, crop and rotate guides etc.)"));
+    g_signal_connect(G_OBJECT(dev->overlay_color.button), "clicked",
+                     G_CALLBACK(_overlay_color_quickbutton_clicked), dev);
+    g_signal_connect(G_OBJECT(dev->overlay_color.button), "button-press-event",
+                     G_CALLBACK(_overlay_color_quickbutton_pressed), dev);
+    g_signal_connect(G_OBJECT(dev->overlay_color.button), "button-release-event",
+                     G_CALLBACK(_overlay_color_quickbutton_released), dev);
+    dt_view_manager_module_toolbox_add(darktable.view_manager, dev->overlay_color.button, DT_VIEW_DARKROOM);
+
+    // and the popup window
+    dev->overlay_color.floating_window = gtk_popover_new(dev->overlay_color.button);
+    gtk_widget_set_size_request(GTK_WIDGET(dev->overlay_color.floating_window), dialog_width/2.0, -1);
```

Comment:
The / 2.0 here is a bit too much, the magenta string is long and gets close to the label. We need a bit more width, especially for some translated languages like French where "overlay color" will probably be "couleur des superpositions" ou "couleur des contrôles".

Reply:
What would you recommend? Should I just leave it at default (dialog_width)?

Reply:
Yes probably as a first approach. I'm not sure using some magic number like 1.2f or 1.3f will be better.

Reply:
ok will do

Reply:
done